modules/trusted-firmware-m/Kconfig.tfm

```
@@ -276,14 +276,53 @@ config TFM_MCUBOOT_DATA_SHARING
 
 endif # TFM_BL2
 
+choice TFM_MODEL
+	prompt "TF-M Firmware Framework model"
+	default TFM_IPC
+	help
+	  The Firmware Framework M (FF-M) provides different programming models
+	  for Secure Partitions.
+
+config TFM_LIBRARY
+	bool "Library Model"
+	help
+	  When enabled this option signifies that the TF-M build use secure
+	  library model instead of PSA API.
```

```suggestion
	  library model instead of the IPC Model.
```

The PSA API is supported in the library model.

---

Strange, but insignificant I guess since this is going to be deleted in a few months time.